org.eclipse.winery.workflowmodeler/src/app/services/jsplumb.service.ts

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2017 ZTE Corporation.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and the Apache License 2.0 which both accompany this distribution,
+ * and are available at http://www.eclipse.org/legal/epl-v10.html
+ * and http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Contributors:
+ *     ZTE - initial API and implementation and/or initial documentation
+ */
+
+import {Injectable} from '@angular/core';
+import {jsPlumb} from 'jsplumb/dist/js/jsplumb.js';
+import {WorkflowNode} from '../model/workflow.node';
+import {BroadcastService} from './broadcast.service';
+import {ModelService} from './model.service';
+
+/**
+ * JsPlumbService
+ * provides all of the operations about jsplumb plugin.
+ */
+@Injectable()
+export class JsPlumbService {
+
+    public jsplumbInstance: any;

```

Comment:
Can you maybe find a class/interface for this variable? Maybe use the console to output it's type?